dsl_parser/parser.py

```diff
@@ -22,13 +22,13 @@
 from dsl_parser.elements import blueprint
 
 
-def parse_from_path(dsl_file_path, resources_base_url=None):
+def parse_from_path(dsl_file_path, resolver, resources_base_url=None):
```

Comment:
cloudify-cli uses `parse_from_path`
https://github.com/cloudify-cosmo/cloudify-cli/blob/master/cloudify_cli/commands/blueprints.py#L40
to do validate, so making resolver mandatory currently breaks `cfy blueprints validate`